scripts/script.js

```diff
@@ -0,0 +1,68 @@
+const groups = [
+    'c_python',
+    'c_excel',
+    'c_web',
+    'c_js',
+    'c_data',
+    'c_aws',
+    'c_draw',
+];
+
+function load_courses(group, filter_s=''){
+    
+    fetch(`./data/${group.slice(2)}_res.json`)
+        .then(res => res.json())
+        .then(data => {
+            const con = document.getElementById("selected_courses");
+            con.innerHTML = `
+                <h2 class="course_title">${data.header}</h2>
+                <p class="course_desc">${data.description}</p>
+                <button class="explore">Explore ${document.getElementById(group).innerText}</button>
+                <ul class="courses_reco">
+                    ${data.courses.filter(course => course.title.toLowerCase().includes(filter_s.toLowerCase())).map(course => (`
+                        <li>
+                            <figure>
+                                <img src="${course.image}" alt="${course.title}">
+                                <figcaption>${course.title}</figcaption>
+                            </figure>
+                            ${course.instructors.map(instructor => `
+                                <h4 class="author">${instructor.name}</h4>
+                            `).join('\n')}
+                            <h3 class="price">$${course.price}</h3>
+                        </li>
```

Comment:
you can refactor these lines of code out to a function that takes a course as an input and return the course HTML.